packages/map-template/src/components/Search/Search.jsx

```diff
@@ -144,8 +146,17 @@ function Search({ onLocationClick, categories, onLocationsFiltered, onSetSize, c
         }
     }, [currentVenueName]);
 
+    /*
+     * React on changes in categories prop.
+     */
+    useEffect(() => {
+        setHasCategories(categories.length > 0);
```

Comment:
Is this state necessary? Could we just use the categories variable instead of hasCategories, or will changing the categories prop not trigger a re-render of the component?

Reply:
I have tried using the `categories` variable instead and it seems to be working as intended. ✅